vulkan/vulkan_native_surface_android.cc

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/vulkan/vulkan_native_surface_android.h"
+#include "third_party/skia/include/gpu/vk/GrVkBackendContext.h"
+
+namespace vulkan {
```

Comment:
It would be great to capture this knowledge in a comment or some other form of documentation (perhaps it is already?).